bin/lib/create.js

```diff
@@ -46,15 +47,15 @@ module.exports.create = function (destinationDir, config, options) {
     var root = path.join(__dirname, '..', '..');
 
     events.emit('log', 'Creating Cordova Windows Project:');
-    events.emit('log', '\tApp Name  : ' + appName);
-    events.emit('log', '\tNamespace : ' + packageName);
-    events.emit('log', '\tPath      : ' + projectPath);
+    events.emit('log', '\tPath: ' + path.relative(process.cwd(), (projectPath || 'CordovaExample')));
```

Comment:
'CordovaExample' will never be substituted here because projectPath must be truthy after the checks at the top of this function.